src/Compilers/CSharp/Portable/FlowAnalysis/LocalDataFlowPass.cs

```diff
@@ -120,6 +120,7 @@ protected virtual int GetOrCreateSlot(Symbol symbol, int containingSlot = 0, boo
         {
             Debug.Assert(containingSlot >= 0);
 
+            if (symbol is null) return -1;
```

Comment:
>if (symbol is null) return -1 [](start = 12, length = 29)

Can we check for `symbol is null` in the caller? It feels like we might be hiding a bug if we allow `null` here.

Reply:
Consider adding `#nullable enable ... #nullable restore` around this method and the `override` in `NullableWalker` to catch cases where `symbol` may be `null`.

---
In reply to: [447947315](https://github.com/dotnet/roslyn/pull/45563#discussion_r447947315) [](ancestors = 447947315)

Reply:
Yeah, putting the check here doesn't seem like the right solution to me. We shouldn't be attempting to get a slot for a non-existant symbol.

---
In reply to: [447954354](https://github.com/dotnet/roslyn/pull/45563#discussion_r447954354) [](ancestors = 447954354,447947315)

Reply:
Moved a check to the correct location, added an Assert here. (Commit 8).

---
In reply to: [447962405](https://github.com/dotnet/roslyn/pull/45563#discussion_r447962405) [](ancestors = 447962405,447954354,447947315)